modules/manually-maintained-credentials-upgrade.adoc

```diff
@@ -5,9 +5,28 @@
 // * installing/installing_gcp/manually-creating-iam-gcp.adoc
 
 [id="manually-maintained-credentials-upgrade_{context}"]
-== Upgrades
+= Upgrading clusters with manually maintained credentials
 
-In a future release, improvements to the Cloud Credential Operator will prevent
-situations where a user might enter an upgrade that will fail because their
-manually maintained credentials have not been updated to match the
-`CredentialsRequests` in the upcoming release image.
+If credentials are added in a future release, the Cloud Credential Operator
+(CCO) `upgradable` status for a cluster with manually maintained credentials
+changes to `false`. For minor releases (for example, from 4.5 to 4.6), this
```

Comment:
The ISG style guide is fairly strict on using parentheses. If there's a way to re-write the parenthetical, that would be ideal.

Reply:
I could do it with commas instead, but thought that was a bit unwieldy. But if a bit unwieldy is preferred over parens, I'll do it ;)